src/screens/Profiles/MediaPages/pages/Episodes/Unit/component.js

```diff
@@ -206,11 +196,9 @@ class Unit extends PureComponent {
               style={styles.webContainer}
               // @TODO: replace with a Kitsu-based link
               source={{ uri: 'https://reminiscent-team.surge.sh' }}
-              onMessage={this.onMessage}
               renderLoading={this.renderLoading}
               renderError={this.renderError}
-              // This ensures `postMessage` has been patched by React-Native
-              injectedJavaScript="window.initializeHulu();"
+              injectedJavaScript={injectedJavaScript}
```

Comment:
If there's a state update while a user is watching a video, does this affect the web view?

Reply:
Nope, not during my tests at least. (Delaying a state change until after video loads)